### PR TITLE
Make `ignore_unreachable` accept jinja templates

### PR DESCRIPTION
**Pull request by audgirka**
Fixes: #4085

### CODE REVIEW

**Review by ssbarnea — CHANGES_REQUESTED** [review]
Please include an use of templated value for the ignore_reachable in the schema test files, so we would not accidentally remove it in the future.

**Review by ssbarnea — CHANGES_REQUESTED** [review]
This is missing a positive test.

File: test/schemas/test/playbooks/ignore-unreachable.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+- name: Test
+  hosts: localhost
+  tasks:
+    - name: Debug
+      ansible.builtin.debug:
+        msg: ignore_unreachable should be a boolean
+      ignore_unreachable: true
```
Comment (ssbarnea) [inline test/schemas/test/playbooks/ignore-unreachable.yml:8]: I think that the bug was about using jinja with ignore_unreachable, poor booleans did  work. Please be sure you include some jinja instead.

Comment (audgirka) [reply test/schemas/test/playbooks/ignore-unreachable.yml:8]: @ssbarnea it's still failing and I'm trying to fix the same

Comment (audgirka) [reply test/schemas/test/playbooks/ignore-unreachable.yml:8]: I thought the PR was in draft!

**Review by ssbarnea — APPROVED** [review]

**Pull request merged**